shopinvader/services/settings.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.base_rest import restapi
+from odoo.addons.component.core import Component
+
+
+class ExportSettingsService(Component):
+    """Shopinvader service to expose allowed settings"""
+
+    _inherit = [
+        "base.shopinvader.service",
+    ]
+    _name = "shopinvader.settings.service"
+    _usage = "settings"
+    _description = __doc__
+
+    def get(self):
```

Comment:
@kevinkhao Can you add the explicit decorator PLZ. 
```suggestion
    @restapi.method(
        [(["/"], "GET")],
        output_param=restapi.CerberusValidator(...)
        auth="public",
    )
    def get(self):
```